tests/cypress/support/authentication.js

```diff
@@ -76,17 +78,7 @@ Cypress.Commands.add("authenticate", (userType, userCredentials) => {
      * This ensures reused sessions maintain these tokens
      * We expect at least three for the id, access, and refresh tokens
      */
-    cy.waitUntil(() =>
-      cy
-        .window()
-        .then(
-          (window) =>
-            Object.keys(window.localStorage).filter((key) =>
-              key.match(
-                /CognitoIdentityServiceProvider.+(refresh|access|id)Token/
-              )
-            ).length === 3
-        )
-    );
```

Comment:
I've updated the authentication call. It was using xpath which is deprecated, and I've removed with waitUtil and key check. Wait until wasn't in the docs from what I could find, and so I really tried to simplify the login process to be safe. That being said, calling [login from the ui is an anti-pattern I guess](https://docs.cypress.io/guides/references/best-practices#Organizing-Tests-Logging-In-Controlling-State)? At least according to cypress. Some information about how we're actually supposed to call it is [here ](https://docs.cypress.io/guides/end-to-end-testing/amazon-cognito-authentication)but I simplified it to work with ours.